CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -0,0 +1,94 @@
+# Changelog 2.0
+
+## 2.0.0 - 2024-04-26
+
+### Added
+
+- Support `LoadBalancerIPMode` in AntreaProxy to implement K8s [KEP-1860](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1860-kube-proxy-IP-node-binding). ([#6102](https://github.com/antrea-io/antrea/pull/6102), [@hongliangl])
+- Add `sameLabels` field support for Antrea ClusterNetworkPolicy peer Namespace selection to allow users to create ACNPs that isolate Namespaces based on their label values. ([#4537](https://github.com/antrea-io/antrea/pull/4537), [@Dyanngg])
+- Add multiple physical interfaces support for the secondary network bridge. ([#5959](https://github.com/antrea-io/antrea/pull/5959), [@aroradaman])
+- Use a Node's primary NIC as the secondary OVS bridge physical interface. ([#6108](https://github.com/antrea-io/antrea/pull/6108), [@aroradaman])
+- Add user documentation for Antrea native secondary network support. ([#6015](https://github.com/antrea-io/antrea/pull/6015) [#6042](https://github.com/antrea-io/antrea/pull/6042), [@jianjuns] [@antoninbas])
+- Add a new versioned API `NetworkPolicyEvaluation` and a new antctl sub-command for querying the effective policy rule applied to particular traffic. ([#5740](https://github.com/antrea-io/antrea/pull/5740) [#6112](https://github.com/antrea-io/antrea/pull/6112), [@qiyueyao])
+
+### Changed
+
+- Multiple deprecated APIs, fields and options have been removed from Antrea.
+  - Remove deprecated v1alpha1 CRDs `Tier`, `ClusterNetworkPolicy`, `NetworkPolicy`, `Traceflow` and `ExternalEntity`. ([#6162](https://github.com/antrea-io/antrea/pull/6162) [#6177](https://github.com/antrea-io/antrea/pull/6177) [#6238](https://github.com/antrea-io/antrea/pull/6238), [@luolanzone] [@hjiajing] [@antoninbas])
+  - Remove deprecated v1alpha2 and v1alpha3 CRDs `ClusterGroups`, `ExternalIPPool`, `ClusterGroup` and `Group`. ([#6049](https://github.com/antrea-io/antrea/pull/6049) [#6239](https://github.com/antrea-io/antrea/pull/6239), [@luolanzone] [@antoninbas])
+  - Remove deprecated `ServiceAccount` field in `ClusterSet` type for Antrea Multi-cluster. ([#6134](https://github.com/antrea-io/antrea/pull/6134), [@luolanzone])
+  - Remove deprecated options `enableIPSecTunnel`,`multicastInterfaces`, `multicluster.enable` and `legacyCRDMirroring`. ([#5158](https://github.com/antrea-io/antrea/pull/5158), [@luolanzone])
+  - Clean up unused code for NodePortLocal and remove the deprecated `nplPortRange` config. ([#5943](https://github.com/antrea-io/antrea/pull/5943), [@luolanzone])
+  - Clean up deprecated APIServices. ([#6002](https://github.com/antrea-io/antrea/pull/6002), [@tnqn])
+- Documentation has been updated to reflect recent changes and provide better guidance to users.
+  - Add upgrade instructions for Antrea v2.0. ([#6261](https://github.com/antrea-io/antrea/pull/6261), [@antoninbas])
```

Comment:
@antoninbas I added a line here  for `Add upgrade instructions for Antrea v2.0 #6261`, please check. Thanks.

Reply:
Looks good, but I would recommend also adding one line at the very beginning of the CHANGELOG, as it is important that users run `antctl upgrade api-storage` before upgrading. Something like this:

```
# Changelog 2.0

*Some deprecated APIs have been removed in Antrea v2.0. Before upgrading, please read these [guidelines](https://github.com/antrea-io/antrea/blob/release-2.0/docs/versioning.md#upgrading-from-antrea-v1-to-antrea-v2) carefully.*

## 2.0.0 - 2024-04-26

### Added
```

Reply:
Added. Thanks.